StreamChatUITestsAppUITests/MockServer/StreamMockServer.swift

```diff
@@ -9,7 +9,7 @@ final class StreamMockServer {
     
     private(set) var server: HttpServer = HttpServer()
     private weak var globalSession: WebSocketSession?
-    var messageDetails: [Dictionary<MessageDetail, String>] = []
+    var messageDetails: [<MessageDetail, String>] = []
```

Comment:
Current implementation fails, something like this works as expected:
```suggestion
    var messageDetails: [[MessageDetail: String]] = []
```

Reply:
Great spot!